WebViewControl/ResourceUrl.cs

```diff
@@ -56,33 +59,46 @@ public partial class ResourceUrl {
         /// <summary>
         /// Supported syntax:
         /// embedded://webview/assembly:AssemblyName;Path/To/Resource
+        /// embedded://webview/assembly:AssemblyName-AssemblyVersion;Path/To/Resource
         /// embedded://webview/AssemblyName/Path/To/Resource (AssemblyName is also assumed as default namespace)
+        /// embedded://webview/AssemblyName-AssemblyVersion/Path/To/Resource
         /// </summary>
         internal static string[] GetEmbeddedResourcePath(Uri resourceUrl) {
             if (ContainsAssemblyLocation(resourceUrl)) {
                 var indexOfPath = resourceUrl.AbsolutePath.IndexOf(AssemblyPathSeparator);
                 return resourceUrl.AbsolutePath.Substring(indexOfPath + 1).Split(new [] { PathSeparator }, StringSplitOptions.None);
             }
             var uriParts = resourceUrl.Segments;
-            return uriParts.Skip(1).Select(p => p.Replace(PathSeparator, "")).ToArray();
+            var assemblyName = uriParts[1].Split(AssemblyVersionSeparator).First();
+            return uriParts.Skip(2).Prepend(assemblyName).Select(p => p.Replace(PathSeparator, "")).ToArray();
```

Comment:
removed version?

Reply:
Yes, this method is meant to return the resource path, which includes the assembly name but is not supposed to include the version.
The version is used only to resolve the assembly.